common/service/config/config.go

```diff
@@ -61,6 +58,8 @@ type (
 		// DynamicConfigClient is the config for setting up the file based dynamic config client
 		// Filepath should be relative to the root directory
 		DynamicConfigClient dynamicconfig.FileBasedClientConfig `yaml:"dynamicConfigClient"`
+		// Domain is the default config for every domain
+		Domain Domain `yaml:"domain"`
```

Comment:
Let's get another opinion about this name. Can you ask Samar his opinion.

Reply:
Looks like I'm missing some context here.  Anything which is overridden by domain usually goes to DynamicConfig rather than static yaml based config.  Our DynamicConfig already has all the infrastructure needed for providing defaults are overriding those defaults per domain.  Let's have a quick discussion before adding this to static config.

Reply:
Use case here is more about various defaults used for domains.  I would rather rename this to DomainDefaults.